src/components/paging.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactElement } from 'react'
+import { useHistory } from 'react-router-dom'
```

Comment:
This looks like a different paging implementation than https://github.com/thegrims/UsTaxes/pull/50

Reply:
Yep I split off #50 to work on this separately, plan on rebasing here after.